lib/aika_web/plugs/api_auth_plug.ex

```diff
@@ -0,0 +1,21 @@
+defmodule AikaWeb.ApiAuthPlug do
+  import Plug.Conn
+  alias Aika.{Repo, Accounts.User}
+
+  def init(default), do: default
+
+  def call(conn, _) do
+    with user_id        <- conn.params["user_id"],
```

Comment:
Might be neater to pattern match for these in the function header and then create a fallback one that returns 401 as well? Also you could query for the user using both id and api_token and then check if anything came back?